xrdp/xrdp_wm.c

```diff
@@ -767,10 +767,19 @@ xrdp_wm_init(struct xrdp_wm *self)
                     list_add_strdup(self->mm->login_values, r);
                 }
 
-                /*
-                 * Skip the login box and go straight to the connection phase
-                 */
-                xrdp_wm_set_login_state(self, WMLS_START_CONNECT);
+                if (self->session->client_info->gfx && !self->mm->egfx_up)
+                {
+                    /* gfx session but have not recieved caps advertise yet,
+                       set flag so we will connect to backend later */
+                    self->mm->gfx_delay_autologin = 1;
```

Comment:
This flag appears to be set but not used.

Reply:
Correct. It's intended for future use, just like the xrdp_egfx.c and xrdp_egfx.h files. It WILL be used once GFX is turned on.